backend/src/controllers/v2/environmentController.ts

```diff
@@ -394,6 +394,11 @@ export const renameWorkspaceEnvironment = async (req: Request, res: Response) =>
     { workspace: workspaceId, environment: oldEnvironmentSlug },
     { environment: environmentSlug }
   );
+  await SecretImport.updateMany(
+    { workspace: workspaceId, 'imports.environment': oldEnvironmentSlug },
+    { $set: { 'imports.$[element].environment': environmentSlug } },
+    { arrayFilters: [{ 'element.environment': oldEnvironmentSlug }] },
```

Comment:
I don't think `arrayFilters` is supported by document db ( we have to stay compatible). Can you update the query to make sure that it is compatible https://docs.aws.amazon.com/documentdb/latest/developerguide/mongo-apis.html

Reply:
I found the `arrayFilters` in one of Amazon's blog articles about Document DB so I guess it works

https://aws.amazon.com/de/blogs/database/new-amazon-documentdb-features-for-aggregations-arrays-and-indexing/